examples/lock-app/lock-common/lock-app.matter

```diff
@@ -2565,6 +2609,16 @@ endpoint 0 {
     ram      attribute clusterRevision default = 1;
   }
 
+  server cluster TimeFormatLocalization {
```

Comment:
Why is this here?  It's not required by spec.

Reply:
Matter root device has this enabled by default:
```<include cluster="Time Format Localization" client="false" server="true" clientLocked="true" serverLocked="false">
                <requireAttribute>HOUR_FORMAT</requireAttribute>
            </include>```

Reply:
Well, then that's wrong in the XML and needs to be fixed.